bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -0,0 +1,37 @@
+{% extends "bare.html" %}
+{% stylesheet 'firefox_new_firstrun' %}
+{% block site_js %}
+  {% javascript 'site' %}
+  {% javascript 'common' %}
+  {% javascript 'firefox_new_firstrun' %}
+{% endblock %}
+
+{% block content %}
+  <div>
+    <div id="scene">
+      <div class="fxaccounts-container">
+        <h1 id="title">{{_('Already Using Firefox?')}}</h1>
+        <div id="left-divider">
+          <p>{{_('Sign in to your account and we\'ll sync the bookmarks, passwords and other great things you\'ve saved to Firefox on other devices.')}}</p>
+          <div>
+            <img src="{{ static('img/firefox/firstrun-horizon/sample.png') }}">
+          </div>
+          <a href="{{ url('firefox.sync') }}">{{_('Learn more about Firefox Accounts')}}</a>
+        </div>
+      <div class="fxaccounts" id="fxa-iframe-config" data-host="{{ settings.FXA_IFRAME_SRC }}" data-mozillaonline-host="{{ settings.FXA_IFRAME_SRC_MOZILLAONLINE }}">
+        <iframe id="fxa" scrolling="no" data-src="{{ settings.FXA_IFRAME_SRC }}signin?utm_campaign=fxa-embedded-form&amp;utm_medium=referral&amp;utm_source=firstrun&amp;utm_content=fx-{{ version }}&amp;entrypoint=firstrun&amp;service=sync&amp;context=iframe&amp;style=chromeless&amp;haltAfterSignIn=true"></iframe>
+        <button id="skip-button">{{_('Skip This Step')}}</button>
+      </div>
+    </div>
+
+      <img id="night-sky" alt=" Night Sky" src="{{ static('img/firefox/firstrun-horizon/night-scene/sky.png') }}">
```

Comment:
I don't think adding alt text for these images makes a whole lot of sense, since they are more just for presentation, and having a screen reader read through these as a list won't really add a lot of meaning to the page. Please use empty `alt=""` attributes.